README.md

```diff
@@ -124,47 +125,56 @@ The development container has been tested on the following platforms:
 
 (2) When building Holoscan SDK on AGX Orin Dev Kit from source please add the option  ```--cudaarchs all``` to the ```./run build``` command to include support for AGX Orin's iGPU.
 
-# Building Applications
+# Building Operators, Applications, and Packages
 
-_Make sure you have installed the [prerequisites](#prerequisites) before attempting to build reference applications._
+> _Make sure you have either launched your [development container](#container-build-recommended) or [set up your local environment](./doc/developer.md#native-build) before attempting to build Holohub components._
 
-Reference applications based on the Holoscan platform may be found under the [applications](./applications/) directory. Reference applications aim to provide a demonstration of the SDK capabilities.
+This repository provides a convenience `run` script to abstract some of the CMake build process below.
 
-This repository provides a convenient `run` script to build and run applications in your development environment.
-To build an application:
+Run the following to list existing components available to build:
 
-```bash
-  ./run build <application>
-```
+  ```bash
+  ./run list
+  ```
+
+Then run the following to build the component of your choice, using either its name or its path:
+
+  ```bash
+  # Build using the component name
+  ./run build <package|application|operator>
+  # Ex: ./run build endoscopy_tool_tracking
 
-When the build is successful you can [run applications](#running-applications).
+  # Build using the component path
+  ./run build ./<pkg|applications|operator>/<name>
+  # Ex: ./run build ./applications/endoscopy_tool_tracking/
+  ```
+
+The build artefacts will be created under `./build/<component_name>` by default to isolate them from other components which might have different build environment requirements. You can override this behavior and other defaults, see `./run build --help` for more details.
```

Comment:
Nitpick suggestion for U.S. spelling convention (though I guess U.K. convention is valid too as we don't enforce a style guide at the moment):

```suggestion
The build artifacts will be created under `./build/<component_name>` by default to isolate them from other components which might have different build environment requirements. You can override this behavior and other defaults, see `./run build --help` for more details.
```